lib/util/user.js

```diff
@@ -70,8 +66,8 @@ export function positionHomeAndWorkFirst (userData, homeName, workName) {
   const homeLocation = savedLocations.find(isHome) || BLANK_HOME
   const workLocation = savedLocations.find(isWork) || BLANK_WORK
 
-  homeLocation.name = homeName || BLANK_HOME.name
-  workLocation.name = workName || BLANK_WORK.name
+  homeLocation.name = homeName || 'Home'
```

Comment:
Are we alright not localizing this?

Reply:
I think we are, since it's really just a fallback, since `intl.formatMessage({ id: 'common.places.home' }),
      intl.formatMessage({ id: 'common.places.work' })` are always passed (the fallback should never be reached). Rly 'Home' and 'Work' can probably just be removed.